src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
```

Comment:
It is enough to create one Lottery object, and then, in a loop with 3 iterations, get a new ball from it and pass it to the println method.
Also, when passing an object to println, there is no need to call their toString() method, it is called under the hood of the print method

Reply:
All changes have been done.